src/rpc/mod.rs

```diff
@@ -465,7 +465,7 @@ where
     module
 }
 
-pub fn openrpc() -> openrpc_types::OpenRPC {
+pub fn openrpc(include: Option<&[&str]>) -> openrpc_types::OpenRPC {
```

Comment:
nit: it would be nice to document this method, e.g. "by default exposes all the available endpoints, if `include` is specified - then only the selected ones".  Not having to read the actual code to understand what a method does goes a long way.